google-cloud-storage/src/test/java/com/google/cloud/storage/ITMultipartUploadHttpRequestManagerTest.java

```diff
@@ -396,7 +398,7 @@ public void sendListPartsRequest_success() throws Exception {
                   + "  <IsTruncated>false</IsTruncated>\n"
                   + "  <Part>\n"
                   + "    <PartNumber>1</PartNumber>\n"
-                  + "    <ETag>\"etag\"</ETag>\n"
+                  + "    <ETag>etag</ETag>\n"
```

Comment:
Can these two changes be moved back to the list PR since they're related to that?

Reply:
Done